plugins/mel_metapage/mel_metapage.php

```diff
@@ -5109,18 +5109,163 @@ public static function IncludeLoader()
     public static function GetSurveyButton() : string {
         $html = '';
         $rc = rcmail::get_instance();
-        $surveyConfig = $rc->config->get('survey', false);
 
-        if ($surveyConfig !== false) {
+        // Vérification si le bouton doit être activé
+        $surveyRaw = $rc->config->get('survey', false);
+        if ($surveyRaw === false) return '';
+
+        $defaults = (array) $rc->config->get('survey_defaults', []);
+        $surveyConfig = ($surveyRaw === true) ? $defaults : $surveyRaw;
+
+        // Filtre par dates
+        if (!self::isInDateWindow($surveyConfig)) {
+            return '';
+        }
+
+        // Affiche-t-on le bouton sur cette page ?
+        if ($surveyConfig !== false && self::shouldDisplaySurvey($surveyConfig)) {
+
+            // Base URL vers la skin
+            $baseUri = rtrim(dirname($_SERVER['SCRIPT_NAME'] ?? ''), '/');
+            $pluginSkinUrl = ($baseUri ? $baseUri : '') . '/plugins/mel_metapage/skins/mel_elastic';
```

Comment:
Tu dois pouvoir te baser sur `$this->local_skin_path()` plutôt que mettre le chemin en dur